src/protos/io/defang/v1/fabric.proto

```diff
@@ -192,6 +192,9 @@ message LogEntry {
   string message = 1;
   google.protobuf.Timestamp timestamp = 2;
   bool stderr = 3;
+  string service = 4;
```

Comment:
I am planning on expanding a few messages to include new fields (see below). This will be added to the loki log labels to make logs more searchable and specific. Can this field be made into another message with service as a field? Then it can be overridden in the future without breaking backward compatibility.

`
message EntityIdentity {
  string platforn = 1;
  string application = 2;
  string environment = 3;
  repeated string services = 4;
}

message LogIdentity {
  string job = 1;
  string instance = 2;
  string etag = 3;
  EntityIdentity entity = 3;
}
`

Reply:
After discussion, will keep a flat message. If we see duplication of the same fields in other messages in the future we will revisit making this a separate message. One of the points against embedded messages was Lio's previous experience with it and find it a bit of a pain having to have additional check for null values of embedded messages.